lang_python.adoc

```diff
@@ -13,130 +13,171 @@ To generate Python modules, Kaitai Struct compiler should be invoked with
 `-t python` option:
 
 [source,shell]
-kaitai-struct-compiler -t python my_spec.ksy
+kaitai-struct-compiler -t python --python-package . my_spec.ksy
 
 This normally generates `my_spec.py` parser module, which is compatible
-with both Python 2 and 3.
+with both Python 2 and 3. It is strongly recommended to use Python 3
+because Python 2 is EOL since 2020-01-01. You can also install
+`python-is-python3` on Ubuntu focal to make `python3` be used via
```

Comment:
What is _focal_, or _Ubuntu focal_?

Reply:
"Focal Fossa" (or "focal" for short) is the codename/internal name of Ubuntu 20.04.

I'm not sure if the `python-is-python3` part is relevant enough to be included here at all - not everyone uses Ubuntu, not every Ubuntu user is on 20.04 yet, and not everyone on 20.04 wants to (or is able to) use `python-is-python3`. I think it's enough to say that Python 2 is no longer supported and Python 3 should be preferred, and let users figure out how to handle Python versions on their system.